Makefile

```diff
@@ -57,7 +57,7 @@ integration-static: .install.ginkgo # It needs to be release so we correctly tes
 
 .install.golangci-lint:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | \
-		BINDIR=$(abspath $(GOTOOLS_BINDIR)) sh -s v1.49.0
+		BINDIR=$(abspath $(GOTOOLS_BINDIR)) sh -s v1.50.1
```

Comment:
nit: could you make the v1.50.1 a constant at the top of the Makefile?

Reply:
Yep, done :+1: